src/App/Fossa/Analyze.hs

```diff
@@ -232,10 +178,60 @@ applyFiltersToProject basedir filters DiscoveredProject{..} =
     Just rel -> do
       applyFilters filters projectType rel projectBuildTargets
 
+runAnalyzers ::
```

Comment:
[minor nit] we should add comment denoting to also update `runAll` (in list targets), when modifying this collection.